tests/kafkatest/tests/core/round_trip_fault_test.py

```diff
@@ -47,32 +55,48 @@ def __init__(self, test_context):
                                      active_topics=active_topics)
 
     def setUp(self):
-        self.zk.start()
+        if self.zk:
+            self.zk.start()
         self.kafka.start()
         self.trogdor.start()
 
     def teardown(self):
         self.trogdor.stop()
         self.kafka.stop()
-        self.zk.stop()
+        if self.zk:
+            self.zk.stop()
 
-    def test_round_trip_workload(self):
+    def remote_quorum_nodes(self):
+        if quorum.for_test(self.test_context) == quorum.zk:
+            return self.zk.nodes
+        elif quorum.for_test(self.test_context) == quorum.remote_raft:
+            return self.kafka.controller_quorum.nodes
+        else: # co-located case, which we currently don't test but handle here for completeness in case we do test it
+            return []
```

Comment:
should this throw an exception? Or does the current code actually work for this case?

Reply:
> throw an exception or does the current code actually work for this case?

The code always needs <Kafka nodes> + <remote quorum nodes>, where the latter are the ZooKeeper or remote Controller Quorum nodes.  If we were to run this test with co-located Raft Quorum controllers then those nodes would be accounted for because they are part of the Kafka nodes, so there is no need to explicitly add them.  So this code is correct in that it return an empty list for that case.  As was indicated in the comment, it's here just in case we ever decide we want to test it.

Reply:
sounds good